opsdroid/loader.py

```diff
@@ -112,7 +112,7 @@ def import_module(config):
 
         if module_spec:
             module = Loader.import_module_from_spec(module_spec)
-            _LOGGER.debug(_("Loaded %s: %s"), config["type"], config["module_path"])
+            _LOGGER.debug(_("Loaded %s: %s"), config["type"], config["module_path"], ".")
```

Comment:
You can't add the dot here because this way you are passing another parameter to this Logging.

You need to add the dot like this:

```python
_LOGGER.debug(_("Loaded %s: %s."), config["type"], config["module_path"])
```